docs/markdown/Gnome-module.md

```diff
@@ -294,7 +294,26 @@ library.
 Returns a custom dependency that can be included when building other
 VAPI or Vala binaries.
 
-*Added 0.36.0*
```

Comment:
I don't think this should be removed, right?

Reply:
Correct. Should be fixed now!